packages/ui-color-picker/src/ColorPreset/index.tsx

```diff
@@ -246,18 +237,25 @@ class ColorPreset extends Component<ColorPresetProps, ColorPresetState> {
     </Tooltip>
   )
   renderSettingsMenu = (color: string, index: number) => (
-    <Menu
-      key={`color-preset-color-${index}`}
-      placement="bottom"
-      trigger={this.renderColorIndicator(color)}
-      mountNode={() => document.getElementById('main')}
+    <Drilldown
       onSelect={this.onMenuItemSelected(color)}
+      trigger={this.renderColorIndicator(color)}
+      key={`color-preset-color-${index}`}
+      rootPageId="root"
+      width="10rem"
+      offsetY="15rem"
     >
-      <Menu.Group label={color} />
-      <Menu.Item value="select">Select</Menu.Item>
-      <Menu.Item value="remove">Remove</Menu.Item>
-    </Menu>
+      <Drilldown.Page withoutHeaderSeparator id="root" renderTitle={color}>
+        <Drilldown.Option value="select" id="select">
+          Select
+        </Drilldown.Option>
+        <Drilldown.Option value="remove" id="remove">
+          Remove
+        </Drilldown.Option>
```

Comment:
Just a suggestion: the options have an own `onOptionClick` prop, probably easier to use here instead of the root `onSelect`.

Reply:
Yeah, I tried, but it is kinda the same effort to use, so I left it as is